docker-compose.yml

```diff
@@ -196,6 +198,17 @@ services:
       - frontend
       - backend
 
+### Smtp ######################################################
+
+  smtp:
+    build: ./mail
+    ports:
```

Comment:
I'd prefer to have the option to specify the ports from the env file to give users the option to pick without us overwriting it after an update.